GenderPayGap.WebUI/Classes/Presentation/SubmissionService.cs

```diff
@@ -484,51 +481,12 @@ public async Task<ReturnViewModel> GetReturnViewModelAsync(long organisationId,
             return result;
         }
 
-        /// <summary>
-        ///     This draft will only return the contents of the draft, if you need to know the user that is currently locking the
-        ///     draft, or when was the draft last written, please use GetReportInfoModelWithDraft
-        /// </summary>
-        /// <param name="organisation"></param>
-        /// <param name="returnModifiedDate"></param>
-        /// <param name="snapshotYear"></param>
-        public async Task<ReportInfoModel> GetReportInfoModelWithLockedDraftAsync(Organisation organisation,
-            DateTime? returnModifiedDate,
-            int snapshotYear)
-        {
-            ReportInfoModel reportInfo = await GetReport_InfoAsync(organisation, returnModifiedDate, snapshotYear);
-            reportInfo.Draft = await GetDraftIfAvailableAsync(organisation.OrganisationId, snapshotYear);
-            return reportInfo;
-        }
-

```

Comment:
This is just a regular unused method (I think it became unused when we refactored something)